### ISSUE TITLE
CMake: Unable to build on VS2019 16.3.0

### ISSUE THREAD

**Issue by H-G-Hristov** [BinomialLLC/basis_universal#61]
I used the Open Folder feature in VS2019 to open the CMake project. The build fails with:

Severity	Code	Description	Project	File	Line	Suppression State
Error	D8021	invalid numeric argument '/Wextra'	

It would be nice if the project features CMakeSettings.json

**Comment by richgel999** [comment]
Yes, the cmake file doesn't work with VS 2019 yet. It's for Linux/OSX. However, there's a .SLN file in there for VS2019 which is what we use. I will fix this eventually, but it's not a super high priority at the moment.
